rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -2511,4 +2511,38 @@ class ReduceSpec extends FlatSpec with Matchers with AppendedClues with Persiste
 
     task.runSyncUnsafe(timeout)
   }
+
+  "term split size max" should "be evaluated (Int16/2 - 1)" in {
+    implicit val errorLog = new ErrorLog[Task]()
+    val p                 = New(bindCount=1)
```

Comment:
FYI @ArturGajowy with `New` positive test is cca. 3 times faster then with sends or receives.